examples/hello-world-udp-proxy/app.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require_once(__DIR__ . '/vendor/autoload.php');
+
+use DevCycle\DevCycleConfiguration;
+use DevCycle\Api\DevCycleClient;
+use DevCycle\Model\DevCycleOptions;
+use DevCycle\Model\DevCycleUser;
+
+$options = new DevCycleOptions(
+    enableEdgeDB: false,
+    bucketingApiHostname: "http://localhost",
```

Comment:
```suggestion
    bucketingApiHostname: "http:/localhost",
```

Only one slash for socket paths

Reply:
I've never seen these commit suggestions before, I'll have to make use of them for our company PRs! I'll resolve this now, though.

Reply:
They're super handy for one-line changes - would immensely recommend.